wpeverest/user-registration
# UR-1295 Fix - Email is not sent to the user and admin when stripe field is conditionally hidden.

### All Submissions:






### Changes proposed in this Pull Request:

Previously, When the stripe gateway field was conditionally hidden and the user registered the account then the email to the user and admin were not delivered. This PR fixes this issue.

### How to test the changes in this Pull Request:

1. Apply conditional logic to stripe gateway field in the form.
2. In frontend hide stripe field conditionally and register the user 
3. Check email delivered to user and admin or not.

### Types of changes:




### Other information:




### Changelog entry

> Fix - Email is not sent to the user and admin when stripe field is conditionally hidden.
